help/help-component/src/bundle/TutorialMessages.properties

```
@@ -67,7 +67,7 @@ From the sidebar, you can pin your most-used sites and access your own private s
 # Pinned Sites Panel (Desktop View Only)
 introToSakai_pinSites.title=Pinned Sites
 introToSakai_pinSites.body=Pinned sites are pinned on the sidebar for easy access.<br /><br />\
-You can pin or unpin a site by clicking on the pin <span class="no-wrap">( <i class="si si-pin-fill" style="font-weight: bold; font-size: 18px;"></i> )</span> beside the site#39;s title.
+You can pin or unpin a site by clicking on the pin <span class="no-wrap">( <i class="si si-pin-fill" style="font-weight: bold; font-size: 18px;"></i> )</span> beside the site&#39;s title.
```

Minor but you should be able to just have `site\'s title` too. Just a little more obvious what the character is. 
```suggestion
You can pin or unpin a site by clicking on the pin <span class="no-wrap">( <i class="si si-pin-fill" style="font-weight: bold; font-size: 18px;"></i> )</span> beside the site\'s title.
```

---

I liked that as well but sadly it won't work :(